wxGLCanvas does not work on Wayland yet, so force the usage of X11
---
--- a/install/darkradiant.desktop.in	2021-08-09 17:21:39.000000000 +0200
+++ -	2021-09-26 16:41:36.762610338 +0200
@@ -11,7 +11,7 @@
 
 # The %f means a single, local, absolute file. Remote files will be
 # first copied in a local temp location and then given to the program:
-Exec=@CMAKE_INSTALL_PREFIX@/bin/darkradiant %f
+Exec=env GDK_BACKEND=x11 @CMAKE_INSTALL_PREFIX@/bin/darkradiant %f
 
 Terminal=false
 
